src/css/vote-CSS.css

```diff
@@ -0,0 +1,109 @@
+html {
+  display: table;
+}
+
+body {
+  padding: 100px;
+  background: linear-gradient(to right, lightblue, lightblue 50%, darkblue 50%);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.test_box {
+  max-width: 100%;
+  height: auto;
+  padding: 20px;
+  border-radius: 25px;
+  background: linear-gradient(to left, lightblue, lightblue 50%, white 50%);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.u-layout {
+  display: block;
+  margin-bottom: 40px;
+  width: 300px;
+  text-align: center;
+}
+h1 {
+  font-size: 30px;
+  font-family: "Varela Round";
+}
+h2 {
+  font-size: small;
+  flex: 0 1 auto;
+  color: grey;
+}
+
+.u-form-sizes {
+  margin-bottom: 20px;
+}
+
+.c-form {
+  font-weight: bold;
+  display: block;
+  height: 40px;
+  width: 300px;
+}
+
+.c-margin-lft-20 {
```

Comment:
So `c-` generally denotes a component, this is a utility so we can use `u-` here instead

Try and keep class names as short as possible, but readable. The more CSS, the larger the file. It all adds up

`u-ml-20`